src/xmerl_c14n.erl

```diff
@@ -0,0 +1,512 @@
+%% esaml - SAML for erlang
+%%
+%% Copyright (c) 2013, Alex Wilson and the University of Queensland
+%% All rights reserved.
+%%
+%% Distributed subject to the terms of the 2-clause BSD license, see
+%% the LICENSE file in the root of the distribution.
+
+%% @doc XML canonocialisation for xmerl
+%%
+%% Functions for performing XML canonicalisation (C14n), as specified
+%% at http://www.w3.org/TR/xml-c14n .
+%%
+%% These routines work on xmerl data structures (see the xmerl user guide
+%% for details).
```

Comment:
Oh I see, all these three files are from the same BSD library. I didn't notice (just happened to look at the macros one because it was short).

Can we give them a clearer path, like `src/thirdparty/esaml/xyz.erl`. Than we have. place to keep their license file as well.

Reply:
They are no longer from the other project, 90% of the code has been been rewritten. It is hard to see at it just looks like I was the author and I just pasted them in, but in fact there are tons of commits after.